packages/website/static/img/favicon/safari-pinned-tab.svg

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" standalone="no"?>
+<!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 20010904//EN"
+ "http://www.w3.org/TR/2001/REC-SVG-20010904/DTD/svg10.dtd">
+<svg version="1.0" xmlns="http://www.w3.org/2000/svg"
+ width="700.000000pt" height="700.000000pt" viewBox="0 0 700.000000 700.000000"
+ preserveAspectRatio="xMidYMid meet">
+<metadata>
+Created by potrace 1.14, written by Peter Selinger 2001-2017
+</metadata>
+<g transform="translate(0.000000,700.000000) scale(0.100000,-0.100000)"
+fill="#000000" stroke="none">
+<path d="M1906 6582 c-56 -14 -77 -26 -132 -74 -39 -35 -61 -67 -116 -168 -43
+-79 -79 -144 -173 -315 -40 -71 -110 -200 -157 -285 -47 -85 -97 -175 -110
+-200 -14 -25 -50 -90 -80 -145 -31 -55 -67 -120 -80 -145 -14 -25 -76 -137
+-138 -250 -62 -113 -124 -225 -138 -250 -122 -220 -182 -333 -182 -341 0 -5
+-4 -9 -10 -9 -5 0 -10 -4 -10 -9 0 -5 -21 -46 -46 -92 -26 -46 -59 -106 -74
+-134 -15 -27 -61 -111 -102 -185 -41 -75 -94 -171 -118 -215 -23 -44 -46 -84
+-50 -90 -24 -31 -48 -134 -45 -195 4 -72 17 -110 80 -220 23 -41 57 -102 75
+-135 18 -33 43 -78 56 -100 47 -86 228 -413 254 -460 15 -27 91 -164 169 -304
+77 -139 141 -255 141 -257 0 -3 11 -21 24 -42 12 -20 38 -64 55 -97 18 -33 42
+-78 55 -100 48 -86 93 -168 374 -675 160 -289 305 -539 322 -557 36 -36 97
+-68 155 -80 22 -5 742 -8 1600 -8 1469 1 1563 2 1612 19 86 29 132 81 233 264
+49 89 90 165 90 167 0 3 11 22 23 43 20 31 113 198 230 412 16 30 47 85 68
+122 22 36 39 69 39 72 0 4 10 23 23 44 13 20 41 69 62 107 21 39 57 104 80
+145 51 91 66 118 185 335 51 94 98 179 105 190 7 11 12 22 11 25 0 3 8 19 20
+35 11 17 37 59 56 95 46 84 97 177 118 215 9 17 31 56 48 88 18 31 46 83 64
+115 17 31 40 73 51 92 11 19 33 60 50 90 16 30 44 80 62 111 133 230 133 292
+0 534 -58 106 -130 236 -152 275 -11 20 -40 71 -63 115 -24 44 -90 166 -148
+270 -58 105 -121 219 -140 255 -20 36 -66 120 -104 187 -37 68 -68 125 -68
+128 0 3 -11 22 -23 43 -13 20 -43 73 -67 117 -39 72 -210 383 -280 510 -15 28
+-117 212 -225 410 -109 198 -211 377 -227 398 -36 45 -99 89 -153 107 -58 20
+-3104 21 -3179 2z"/>
+</g>
+</svg>
```

Comment:
https://github.com/typescript-eslint/typescript-eslint/pull/6147#issuecomment-1336123675

@JamesHenry - because Apple told me to.

https://developer.apple.com/library/archive/documentation/AppleApplications/Reference/SafariWebContent/pinnedTabs/pinnedTabs.html

Reply:
Ah sorry I meant more, how come it's just the shape without the TS?

This is the current result (compared to apple and YouTube):

<img width="145" alt="image" src="https://user-images.githubusercontent.com/900523/205434755-5083d0c4-494b-44de-a3cd-735ab22ade6c.png">

If we can apply the TS in the middle it won't quite be the same as the actual logo but maybe still preferable?

Reply:
Oh I don't know - and had assumed this wasn't possible. I've never used this feature myself 😅. Do you know how to edit the svg to make that better? Maybe a good followup issue for a firsttime contributor if none of us can quickly do it?

Reply:
If I use this generator it does a good job when I select the second option:

<img width="1232" alt="image" src="https://user-images.githubusercontent.com/900523/205434821-dc16b19e-d093-49c3-891f-044a734f8011.png">

https://realfavicongenerator.net/

(I didn't change to the blurple color in this example)

Reply:
Haha I'd used the same generator! It's a good one. Can you commit what you got out? I can try to tweak it for coloring.

Reply:
Pushed! And I applied the blurple already so hopefully should render correctly in safari, I'll check the netlify preview once it's done

Reply:
<img width="141" alt="image" src="https://user-images.githubusercontent.com/900523/205435437-4f7fe67e-f1e1-452e-90b9-2387c0e37275.png">

:shipit: